lib/arena.rb

```diff
@@ -1,2 +1,31 @@
 class Arena
+  attr_accessor :name, :gladiators
+  def initialize (name)
+    @gladiators = []
+    @name = name.capitalize
+  end
+
+  def add_gladiator (gladiator_name)
+    if @gladiators.length<2
+      @gladiators << gladiator_name
+    else
```

Comment:
dont think you need this `else`, also if you only have an `if` condition, Ruby allows you to do this with one line, i.e.:
`@gladiators << gladiator_name if @gladiators.length < 2`